web/src/components/databaseDashboard/index.tsx

```diff
@@ -3,43 +3,125 @@ import {observer} from "mobx-react-lite";
 
 import styles from "./databaseDashboard.module.scss";
 
+import {useAppState} from "src/state/providers";
 import {useDatabaseState} from "src/state/providers";
+import {DatabaseTab} from "src/state/models/database";
+
+import {
+  DocsQuickstartIcon,
+  DocsTutorialIcon,
+  DocsEasyEdgeDBIcon,
+  DocsIcon,
+} from "src/ui/icons/docs";
 
 import Button from "src/ui/button";
 
+import {
+  HeaderDatabaseIcon,
+} from "src/ui/icons";
+
 export default observer(function DatabaseDashboard() {
   const dbState = useDatabaseState();
+  const appState = useAppState();
+
+  if (dbState.migrationId === null) {
+    return <FirstRunDashboard />;
+  }
 
   return (
     <div className={styles.dashboard}>
-      Dashboard
-      <br />
-      {dbState.migrationId === null ? <FirstRunCard /> : null}
+      <div className={styles.dbName}>
+        <HeaderDatabaseIcon/>
+        <span>{dbState.name}</span>
+      </div>
+
+      <div className={styles.buttons}>
+        <Button
+            label="Open REPL"
+            size="large"
+            style="square"
+            onClick={async () => {
+              appState.currentPage!.setCurrentTabId(DatabaseTab.Repl)
+            }}
+          ></Button>
+
+        <Button
+            label="Browse Schema"
+            size="large"
+            style="square"
+            onClick={async () => {
+              appState.currentPage!.setCurrentTabId(DatabaseTab.Schema)
+            }}
+          ></Button>
+
+        <Button
+            label="Browse Data"
+            size="large"
+            style="square"
+            onClick={async () => {
+              appState.currentPage!.setCurrentTabId(DatabaseTab.Data)
+            }}
+          ></Button>
+      </div>
+
+      <div className={styles.docButtons}>
+        <a href="https://www.edgedb.com/docs/guides/quickstart">
```

Comment:
Should we have these links open in a new tab?

Reply:
Yeah, good idea. Can you push a fix?